default_config.ini

```diff
@@ -0,0 +1,8 @@
+[pika]
```

Comment:
I could, but doesn't each fmn.<project> have there own config in /fedmsg.d/fmn.py ?